sdk/tests/deqp/functional/gles3/es3fTextureFormatTests.js

```diff
@@ -1125,6 +1125,9 @@ es3fTextureFormatTests.genTestCases = function() {
         ['gl.COMPRESSED_RGBA8_ETC2_EAC', 'etc2_eac_rgba8', tcuCompressedTexture.Format.ETC2_EAC_RGBA8],
         ['gl.COMPRESSED_SRGB8_ALPHA8_ETC2_EAC', 'etc2_eac_srgb8_alpha8', tcuCompressedTexture.Format.ETC2_EAC_SRGB8_ALPHA8]
     ];
+    if (!gluTextureUtil.enableCompressedTextureES30()) {
+        etc2Formats = [];
```

Comment:
Also, here:

debug('Skipping testing ETC2 texture formats because WEBGL_compressed_texture_es3_0 is unsupported');